teuthology/provision/openstack.py

```diff
@@ -132,14 +132,16 @@ def create(self, num, os_type, os_version, arch, resources_hint):
         flavor = self.flavor(resources_hint['machine'], arch)
         keypair = config['openstack']['keypair'] or 'teuthology'
         worker_group = config['openstack']['worker_group'] or 'teuthology-worker'
+        user_data = self.user_data
+        self.run("cat " + str(user_data))
```

Comment:
I would prefer to use python instead of specific wrapper scripts, what about this:
```
with open(self.user_data) as f:
    print('Using data file: %s' % self.user_data)
    print(f.read())
```

Reply:
Also, It would be probably better to put to `init_user_data` method right after ```open(self.user_data, 'w').write(user_data)```